### DISCUSSION TITLE
❌ Step 5 finish

### DISCUSSION THREAD

**Discussion by sujanbimali9** [skills/.github#414] [Code with Copilot]
failed to complete
![Screenshot from 2023-11-04 19-25-11](https://github.com/skills/.github/assets/137367456/cb4d5477-31ee-4bc8-acc1-dae058776760)

**Comment by R1shabh-Gupta** [comment]
Did you find any solution?

**Comment by vitamin2025** [comment]
Hello, I faced the same problem, and it seems like one of the bot's numbering problems. But I got a solution for this!

So, basically the bot had to declare that we have finished step 5, instead it declared that we have finished "step X". As you can see below:

![image](https://github.com/skills/.github/assets/72799488/43b88b65-7612-4e5b-a153-68dcf4ab3d05)

**The Solution for this is** 

Step 1. Go to your repo and open -step.txt folder.
You can locate this folder in .github > steps 

Step 2. Change X to 5 as shown below:

![image](https://github.com/skills/.github/assets/72799488/76b59128-d1a3-42b6-a1b3-bb5207fd3a00)


Step 3. Now push it and wait for 1 min, it will fix the problem!
